monai/networks/utils.py

```
@@ -863,6 +875,82 @@ def convert_to_torchscript(
+        loaded_module = exported.module()
+        loaded_module.to(device)
+        model.to(device)
+
+        with torch.no_grad():
+            set_determinism(seed=0)
+            torch_out = ensure_tuple(model(*verify_args))
+            set_determinism(seed=0)
+            export_out = ensure_tuple(loaded_module(*verify_args))
+            set_determinism(seed=None)
+
+        for r1, r2 in zip(torch_out, export_out):
```

Ruff (0.15.6)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)